docs/docs/install.md

```diff
@@ -25,22 +25,20 @@ clutter.
 
 ## Base Conda Environment
 
-If you already have a conda-forge base environment, then you can skip this
+If you already have a conda-forge or micromamba base environment, then you can skip this
 step. However, you should consider setting the "solver" in the base environment
-to use libmamba. This will greatly speed up the dependency resolution
-calculation. Once you decide on the install prefix for your overall conda
-environment you can use the included bootstrap script. For this example, we
-will use `/opt/conda` as the path to the conda base installation. Now run the
-bootstrap script:
-
-    $> ./tools/bootstrap_base "/opt/conda"
-
-This bootstrap will install a base system with the conda-forge channel set to
-the default and using the libmamba solver. You can now source the conda
-initialization file and activate this base environment:
-
-    $> source /opt/conda/etc/profile.d/conda.sh
-    $> conda activate base
+to use `libmamba`. To use `libmamba` solver see
+[this](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) article.
+This will greatly speed up the dependency resolution
+calculation.
+
+For new installation run following command to install miniforge
+```
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
```

Comment:
Oh, I didn't notice the script will also update base environment. I thought the script was mainly for creating a default configuration, but default is good now. So I updated documentation to use standard method. My purpose was to reduce complexity.

I think we can either keep the using the script, (this would be preferred if we want to add more stuff in it), or update documentation and let user update base manually.

Reply:
Ok, I personally use the bootstrap script so that I don't have to do all the steps manually :-)   But if you think the user would rather copy / paste the curl command, then please add the commands to update base to the docs as well (since the downloaded installer may have out of date packages).  Either way, let's leave the bootstrap script in the source tree since it saves time when quickly setting up a new base environment.  I also use it in the unit tests to work around problems with the github actions for setting up a conda base.